srtcore/core.h

```diff
@@ -683,7 +683,7 @@ class CUDTGroup
     bool applyGroupTime(time_point& w_start_time, time_point& w_peer_start_time)
     {
         using srt_logging::mglog;
-        if (m_tsStartTime == steady_clock::zero())
+        if (srt::sync::is_zero(m_tsStartTime))
```

Comment:
You could do locally `using namespace srt::sync` to keep the calls short.

Reply:
Added `using srt::sync::is_zero` statement.

Reply:
Ok, even better.